caddy/caddy.go

```diff
@@ -118,6 +126,17 @@ func (f *FrankenPHPApp) UnmarshalCaddyfile(d *caddyfile.Dispenser) error {
 				}
 
 				f.NumThreads = v
+			case "max_threads":
+				if !d.NextArg() {
+					return d.ArgErr()
+				}
+
+				v, err := strconv.Atoi(d.Val())
```

Comment:
Maybe should we use `ParseUint()` to prevent negative values?
The "problem" already exists with `NumThreads`.